tests/unit/api.py

```diff
@@ -1475,4 +1499,4 @@ def test_node_power_cycle(self, db):
         api.project_create('anvil-nextgen')
         api.node_register('node-99', 'ipmihost', 'root', 'tapeworm')
         api.project_connect_node('anvil-nextgen', 'node-99')
-        api.node_power_cycle('node-99')
+        api.node_power_cycle('node-99')
```

Comment:
@henn That little arrow is simply complaining that there is no newline at the end of file.
@xuhang57 A simple enter should do the trick.